### PR TITLE
Coffee house

### PR DESCRIPTION
**Pull request by GevinduH**
Changes Made:
- Created a website for a coffee shop using HTML and CSS.

Details:
- Added  index.html.
- Added coffee.html
- Added  style.css file for styling.
- Added images from images folder.
- HTML document is styled with css.
- Added links to the webpage.
- Added a nav bar and a footer to both pages

### CODE REVIEW

File: docs/coffee.html [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+                    fill="#403F3D" />
+                </svg>
+              </a>
+            </div>
+            <div class="nav">
+              <ul class="nav list">
+                <li class="nav-list">
```
Comment (vieraboschkova) [inline docs/coffee.html]: nav list and nav-list could be a bit misleading perhaps. A common name to add is "item", so nav-item or list-item

File: docs/coffee.html [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+              </a>
+            </div>
+          </div>
+        </div>
+      </header>
+      <main class="menu">
+        <div class="menu container">
+          <div class="menu-offer">
+            <div>
+              <h2>
```
Comment (vieraboschkova) [inline docs/coffee.html:144]: Would be great to have" `<h1>`" on this page as well

File: docs/coffee.html [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+                Behind each of our cups <br />hides an
+                <span class="accent" style="color: var(--accent)">
+                  amazing surprise
+                </span>
+              </h2>
+            </div>
+            <div class="tab-item">
```
Comment (vieraboschkova) [inline docs/coffee.html]: You may may to have another go at the class names - tab-item is actually the full div with all items, so something closer to tab-list. Items 1,2,3 seem to have exactly the same styling, you could just use "tab-item" for each one of them.

File: docs/coffee.html [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+                Dessert
+              </button>
+            </div>
+          </div>
+          <div class="menu-grid">
+            <div class="preview coffee-card">
+              <img
+                src="./images/coffee-1.jpg"
+                alt="Photo of a Irish coffee"
+                class="preview image1" />
```
Comment (vieraboschkova) [inline docs/coffee.html]: Same as with tab items - only one class for all is enough - "preview image"

File: docs/coffee.html [hunk 5]
```diff
@@ -0,0 +1,432 @@
...
+              <h3 class="name-and-the-price-of-the-coffee">$6.50</h3>
+            </div>
+          </div>
+        </div>
+      </main>
+      <footer id="contacts">
+        <div class="footer container">
+          <div class="offer">
+            <div class="footer-Text">
+              <h2 class="accent" id="Footer-Text1">
```
Comment (vieraboschkova) [inline docs/coffee.html]: Is there a reason for the h2 to have ids?

File: docs/index.html [hunk 6]
```diff
@@ -0,0 +1,551 @@
...
+                  <rect x="0.5" y="0.5" width="59" height="59" rx="29.5" />
+                  <path
+                    d="M36.5 30H24M24 30L30 24M24 30L30 36"
+                    stroke-linecap="round"
+                    stroke-linejoin="round" />
+                </svg>
+              </button>
+            </div>
+            <div class="row-slider">
+              <div class="image1">
```
Comment (vieraboschkova) [inline docs/index.html]: image1,2,3 - like in the coffee.html - reduce to one class and reuse. The fewer reusable classes, the easier it is to work on the project later.

File: docs/index.html [hunk 7]
```diff
@@ -0,0 +1,551 @@
...
+            <img
+              src="./images/mobile-screens.jpg"
+              alt="Picture of two phones that has opened the mobile application on the screen" />
+          </div>
+        </div>
+      </section>
+      <footer id="contacts">
+        <div class="footer container">
+          <div class="offer">
+            <div class="footer-Text">
```
Comment (vieraboschkova) [inline docs/index.html]: Make sure to stick to the same convention when choosing class names

File: docs/style.css [hunk 8]
```diff
@@ -0,0 +1,565 @@
+:root {
```
Comment (vieraboschkova) [inline docs/style.css:1]: Good job on using the variables!

File: docs/style.css [hunk 9]
```diff
@@ -0,0 +1,565 @@
...
+    line-height: 150%;
+    color: var(--dark);
+    cursor: pointer;
+}
+
+.btn:hover {
+    color:var(--light);
+}
+
+.btn.menu {
```
Comment (vieraboschkova) [inline docs/style.css]: you use btn class and button class. Could they be reused? or maybe have a different names so it is clear which button is it?

File: docs/style.css [hunk 10]
```diff
@@ -0,0 +1,565 @@
...
+    text-align: justify;
+}
+
+.image.Frappuccino, .image.Macchiato, .image.Ice-coffee {
+    width: 480px;
+    height: 480px;
+    object-fit: contain;
+}
+
+.image1, .image2, .image3 {
```
Comment (vieraboschkova) [inline docs/style.css]: Look at the comments to html files, I think one class should be universal enough

File: docs/style.css [hunk 11]
```diff
@@ -0,0 +1,565 @@
...
+    display: flex;
+    gap: 40px;
+}
+
+.column1, .column2 {
+    display: grid;
+    gap: 40px;
+}
+
+.about-picture {
```
Comment (vieraboschkova) [inline docs/style.css:384]: If there is a way, try grouping classes by sections, so its easier to understand how they are connected. You can add some comments to explain it as well if necessary

File: docs/style.css [hunk 12]
```diff
@@ -0,0 +1,565 @@
...
+    width: 660px;
+    height: 590px;
+}
+
+.about-picture.picture-2 {
+    width: 660px;
+    height: 430px;
+}
+
+.about-picture.picture-3 {
```
Comment (vieraboschkova) [inline docs/style.css:398]: There might be a way to define the picture sizes using grid, but we can leave it for the next part of the project.

File: docs/style.css [hunk 13]
```diff
@@ -0,0 +1,565 @@
...
+    margin-top: -10px;
+}
+
+.menu.container {
+    height: 1328px;
+    text-align: center;
+}
+
+.menu-offer {
+    /* margin-top: -20px; */
```
Comment (vieraboschkova) [inline docs/style.css]: Remove?

File: docs/style.css [hunk 14]
```diff
@@ -0,0 +1,565 @@
...
+
+.tab-item {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 16px;
+    justify-content: space-between;
+}
+
+.item1, .item2, .item3 {
```
Comment (vieraboschkova) [inline docs/style.css]: Comment like in html file, probably unnecessary to have 3 classes. The less the better.

File: docs/style.css [hunk 15]
```diff
@@ -0,0 +1,565 @@
...
+    margin-top: -10px;
+}
+
+#hero-image {
+    max-width: 1360px;
+    height: 644px;
+    border-radius: 40px;
+}
+
+#Footer-Text1 {
```
Comment (vieraboschkova) [inline docs/style.css]: Make sure to follow a naming convention, also towards upper/lower case naming.

**Review by vieraboschkova — CHANGES_REQUESTED** [review]
Good job on the layout!  Especially on the grid and flex usage and css variables!

Tiny fixes:
- some changes in class names
- footer margin
- footer conainters closer to the original figma template

additionally, it would be good to add a readme file to explain what is the content of this repo.

File: docs/coffee.html (hunk 1 above)
Comment (GevinduH) [reply docs/coffee.html]: Fixed

File: docs/coffee.html (hunk 3 above)
Comment (GevinduH) [reply docs/coffee.html]: Fixed.

File: docs/coffee.html (hunk 4 above)
Comment (GevinduH) [reply docs/coffee.html]: Fixed

File: docs/coffee.html (hunk 5 above)
Comment (GevinduH) [reply docs/coffee.html]: Apparently no. So I changes the Ids into classes.

File: docs/index.html (hunk 6 above)
Comment (GevinduH) [reply docs/index.html]: Fixed

File: docs/style.css (hunk 9 above)
Comment (GevinduH) [reply docs/style.css]: I change the btn class to button class. If we want to know which button is that, we can get an idea from the second class. As an example Here the first class is "button" and the second one is "menu". So there are only two buttons for menu, one is in the nav bar, other one is in the section which has the id of "hero" .

File: docs/style.css (hunk 13 above)
Comment (GevinduH) [reply docs/style.css]: Done.

File: docs/coffee.html (hunk 5 above)
Comment (vieraboschkova) [reply docs/coffee.html]: Remove id if not needed and check the class names

File: docs/style.css [hunk 16]
```diff
@@ -0,0 +1,557 @@
...
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 0px;
+}
+
+.menu-grid {
+    margin-top: 30px;
+}
+
+.name-and-the-price-of-the-coffee {
```
Comment (vieraboschkova) [inline docs/style.css]: This name seems very specific. Will it not be later used for the price of tea and dessert? Perhaps something like "price-data" or just "price-text"?

File: docs/style.css [hunk 17]
```diff
@@ -0,0 +1,557 @@
...
+
+.tab-list {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 16px;
+    justify-content: space-between;
+}
+
+.item{
```
Comment (vieraboschkova) [inline docs/style.css]: Did you use prettier for css? Would be great to have the same spacing  and formatting everywhere:
".classname {" or ".classname{"

File: docs/style.css [hunk 18]
```diff
@@ -0,0 +1,557 @@
...
+.item:hover {
+    background-color: var(--container);
+    color: var(--light);
+}
+
+.item:hover .icon {
+    background: #E1D4C9;
+}
+
+.item1:active .icon {
```
Comment (vieraboschkova) [inline docs/style.css]: item1 still needed?

File: docs/style.css [hunk 19]
```diff
@@ -0,0 +1,557 @@
...
+
+.social-buttons:hover .social-btn-elements {
+    fill: var(--dark);
+}
+
+.download-text {
+    padding-top: 100px;
+}
+
+.Footer-Text1 {
```
Comment (vieraboschkova) [inline docs/style.css]: Please rename the classes to fit to your convention

File: docs/index.html (hunk 7 above)
Comment (vieraboschkova) [reply docs/index.html]: I thin we could also change it to something like footer-text__title, depending on your convention, or BEM, but lets leave it like that for now.

File: docs/style.css (hunk 9 above)
Comment (vieraboschkova) [reply docs/style.css]: General rule of a thumb - style using classes not ids, leave ids for interaction with JS.

File: docs/style.css (hunk 16 above)
Comment (GevinduH) [reply docs/style.css]: Fixed.

File: docs/style.css [hunk 20]
```diff
@@ -0,0 +1,557 @@
...
+    background: var(--light);
+    padding: 12px 20px;
+    width: 200px;
+    height: 64px;
+    border: 1px solid var(--dark);
+    border-radius: 100px;
+    gap: 8px;
+}
+
+footer#contacts {
```
Comment (vieraboschkova) [inline docs/style.css]: Over here would be better to add a class name, overwritting the styles could be hard later. Leave it as is, but for the future, try not to select styling by ids.

File: docs/style.css (hunk 17 above)
Comment (GevinduH) [reply docs/style.css]: Not using prettier specifically for css. It formats HTML but not the css. I'd like know the way to configure it.

File: docs/style.css (hunk 18 above)
Comment (GevinduH) [reply docs/style.css]: Sorry. It was a mistake. Now I fixed it.

File: docs/index.html [hunk 21]
```diff
@@ -0,0 +1,550 @@
...
+                    stroke-linejoin="round" />
+                </svg>
+              </button>
+            </div>
+            <div class="row-slider">
+              <div class="slider-image">
+                <img
+                  src="./images/S'mores-Frappuccino.png"
+                  alt="Picture of a S'mores Frappuccino"
+                  class="image Frappuccino" />
```
Comment (vieraboschkova) [inline docs/index.html:221]: We will have to later refactor the "Frappuccino" class

File: docs/style.css (hunk 19 above)
Comment (GevinduH) [reply docs/style.css]: Done.

File: docs/style.css (hunk 17 above)
Comment (vieraboschkova) [reply docs/style.css]: Check this SO answer, maybe it helps:
https://stackoverflow.com/questions/54092631/vs-code-how-to-auto-format-css

File: docs/style.css (hunk 18 above)
Comment (vieraboschkova) [reply docs/style.css]: I cant see the fix yet

File: docs/style.css [hunk 22]
```diff
@@ -0,0 +1,557 @@
...
+    width: 204px;
+    margin-top: -10px;
+}
+
+#hero-image {
+    max-width: 1360px;
+    height: 644px;
+    border-radius: 40px;
+}
+
```
Comment (vieraboschkova) [inline docs/style.css]: 1 empty line at the end of the file

Comment (GevinduH) [reply docs/style.css]: Fixed.

File: docs/style.css (hunk 11 above)
Comment (vieraboschkova) [reply docs/style.css:384]: For the next project, its gonna be good to start grouping them from the beginning, or maybe have in separate files. For this one is ok.

**Review by vieraboschkova — APPROVED** [review]
Thank you for the fixes!

**Pull request merged**